### ISSUE TITLE
Make maxWaitTimeAfterFirstMessage configurable for Servicebus Receive function

### ISSUE THREAD

**Issue by peterzeller** [Azure/azure-sdk-for-go#15963]
### Feature Request

The field `maxWaitTimeAfterFirstMessage` in https://github.com/Azure/azure-sdk-for-go/blob/47db7e2935fc81ededd911e1b268c8729d5af38d/sdk/messaging/azservicebus/receiver.go#L150 should be made public so that I can use batch receives with a latency below 1 second.

**Comment by richardpark-msft** [comment]
Let me see what the team thinks, but I agree it'd be useful.

**Comment by richardpark-msft** [comment]
This spurred some interesting discussion in our team, so thank you for submitting it!

In the immediate term, I've changed the the default wait time after first message to 20ms, rather than 1 second. We've got some good practical evidence (given that this is the value that the .NET stack uses) that this should work better in practice.

This will be released in the next beta (coming out in the next few days).

Longer term, we're still discussing if the underlying algorithm so we won't be exposing the setting publicly.
